tests/conftest.py

```diff
@@ -228,134 +235,90 @@ def create_resource(kube_client, kube_ns, resource_type, manifest, delete_first=
-            }
-        )
-
-    pod_manifest = V1Pod(
-        metadata={
-            "name": pod_name,
-            "labels": {"component": "hub", "hub-name": pod_name},
-        },
-        spec=V1PodSpec(
-            volumes=volumes,
-            containers=[
-                {
-                    "image": "jupyterhub/jupyterhub:1.3",
-                    "name": "hub",
-                    "volumeMounts": volume_mounts,
-                    "args": [
-                        "jupyterhub",
-                        "-f",
-                        "/etc/jupyterhub/jupyterhub_config.py",
-                    ],
-                    "env": [{"name": "PYTHONUNBUFFERED", "value": "1"}],
-                    "readinessProbe": {
-                        "tcpSocket": {
-                            "port": 8081,
-                        },
-                        "periodSeconds": 1,
-                    },
-                }
-            ],
-        ),
-    )
-    pod = create_resource(kube_client, kube_ns, "pod", pod_manifest)
-    return wait_for_pod(kube_client, kube_ns, pod_name)
-
-
-@pytest.fixture(scope="session")
-def hub_pod(kube_client, kube_ns):
-    """Create and return a pod running jupyterhub"""
-    return create_hub_pod(kube_client, kube_ns)
-
-
-@pytest.fixture
-def hub(hub_pod):
-    """Return the jupyterhub Hub object for passing to Spawner constructors
-
-    Ensures the hub_pod is running
-    """
-    return Hub(ip=hub_pod.status.pod_ip, port=8081)
-
-
-@pytest.fixture(scope="session")
-def hub_pod_ssl(kube_client, kube_ns, ssl_app):
-    """Start a hub pod with internal_ssl enabled"""
-    # load ssl dir to tarfile
-    buf = io.BytesIO()
-    tf = tarfile.TarFile(fileobj=buf, mode="w")
-    tf.add(ssl_app.internal_certs_location, arcname="internal-ssl", recursive=True)
-
-    # store tarfile in a secret
-    b64_certs = base64.b64encode(buf.getvalue()).decode("ascii")
-    secret_name = "hub-ssl-secret"
-    secret_manifest = V1Secret(
-        metadata={"name": secret_name}, data={"internal-ssl.tar": b64_certs}
-    )
-    create_resource(kube_client, kube_ns, "secret", secret_manifest)
-
-    name = "hub-ssl"
-
-    service_manifest = V1Service(
-        metadata=dict(name=name),
-        spec=V1ServiceSpec(
-            type="ClusterIP",
-            ports=[V1ServicePort(port=8081, target_port=8081)],
-            selector={"hub-name": name},
-        ),
-    )
-
-    create_resource(kube_client, kube_ns, "service", service_manifest)
-
-    return create_hub_pod(
-        kube_client,
-        kube_ns,
-        pod_name=name,
-        ssl=True,
-    )
-
-
-@pytest.fixture
-def hub_ssl(kube_ns, hub_pod_ssl):
-    """Return the Hub object for connecting to a running hub pod with internal_ssl enabled"""
-    return Hub(
-        proto="https",
-        ip=f"{hub_pod_ssl.metadata.name}.{kube_ns}",
-        port=8081,
-        base_url="/hub/",
-    )
+# Base create_moneypenny_pod on this et seq
```

Comment:
Reminder to delete all the commented-out code before merging.

Reply:
tests are very far from being ready, but I will

Reply:
Done